src/components/display/Text.tsx

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Why not put these in shared as well? Not sure what the display subdir communicates